dataland-frontend/src/components/forms/parts/elements/basic/MultiSelectFormElementBindData.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <MultiSelect
+    v-model="selections"
+    :options="options"
+    :placeholder="placeholder"
+    :show-toggle-all="false"
+    :class="innerClass"
+    :optionValue="optionValue"
+    :optionLabel="optionLabel"
+    :maxSelectedLabels="3"
+  />
+  <!--
+    Note: It is required to set the id of this div to the FormKit node Id to allow the checkCustomInputs methods
+    in the validationUtils.ts file to scroll to this component when an error is detected. This is because the FormKit
+    List type does not create a wrapper component on its own.
+  -->
+  <div :id="formkitMultiSelectFormElementBindData?.node?.props?.id || undefined">
+    <FormKit
+      type="list"
+      ref="formkitMultiSelectFormElementBindData"
+      :validation-label="validationLabel"
+      :validation="validation"
+      :name="name"
+      v-model="selections"
+      outer-class="hidden-input"
+      :ignore="ignore"
```

Comment:
My understanding from a call with Marcin on Friday was, that the whole point of creating `MultiSelectFormBindData` instead of using the already existing `MultiSelectForm` was to set the "ignore" to "true" without having to implement the "ignore" prop in the devtools.
So why is this then a dynamic prop?  Should this be hardcoded to "true" ?

Reply:
I've just checked and the default for "ignore" is set to `false`.
So just a question:   Can't we use the `MultiSelectForm` we already have and just set "ignore" to `true` when we use it in the Procurement-Type-component?

Reply:
Whole point of creating MultiSelectFormBindData is to keep the state of the data in the parent and not directly in the component. :ignore is because sometimes, for example, here we do not want formkit to send data from multiselect because it is only used for presentation, while the actual data is in the hidden input

Reply:
Yes you are right.
And the `ignore` field is gone anyways, so I am closing this conversation now.